workers/recon.rb

```diff
@@ -50,9 +52,12 @@ class Recon
     #   The params could not be validated or coerced.
     #
     def perform(params)
-      kwargs  = validate(params)
-      values  = kwargs[:scope].map(&Ronin::Recon::Value.method(:parse))
-      ignore  = kwargs[:ignore].map(&Ronin::Recon::Value.method(:parse))
+      kwargs            = validate(params)
+      values            = kwargs[:scope].map(&Ronin::Recon::Value.method(:parse))
+      ignore            = []
+      if (ignore_kwargs = kwargs[:ignore])
```

Comment:
Add a newline before the `if` block.

Also, I prefer to use the return value of `if`/`else` blocks.

```ruby
value = if foo
          ...
        else
          ...
        end
```